metaci/build/templates/build/build_list.html

```diff
@@ -60,12 +60,18 @@
         <div class="slds-truncate" title="{{ build.branch.name }}">{{ build.branch.name }}</div>
       </td>
       <td data-label="Commit">
-        <div class="slds-truncate" title="{{ build.commit }}">{{ build.commit }}</div>
+        <div class="slds-truncate" title="{{ build.get_commit }}">{{ build.get_commit }}</div>
```

Comment:
We can remove the `title` attribute (which controls the tooltip) in the cases where it's the same thing that is shown in the actual table cell.

Do you think it would make sense to make all these build listings include a common template for the table so we don't have to make these changes in so many places?

Reply:
Yes, a `build_table.html` template would be a nice thing right now :D

Reply:
I'll leave it up to you whether to do that now or in a separate PR.